docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     expose:
       - "80"
     ports:
-      - "81:80"
+      - "80:80"
```

Comment:
what does this change have to do with "Generate api typescript models" ? :)

Reply:
nothing :-(